### PR TITLE
Editorial: add more folks to Acknowledgements

### LINKED ISSUE #13: Add an Acknowledgment section
As mentionned by Alastor, it would be good to add an Acknowledgement section.

### LINKED ISSUE #15: Add an Acknowledgment section
As mentionned by Alastor, it would be good to add an Acknowledgement section.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes w3c/audio-session-detach#13

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -136,4 +136,6 @@ navigator.audioSession.onstatechange = () => {
 
 # Acknowledgements # {#acknowledgements}
 
-TODO
+The Working Group acknowledges the following people for their invaluable contributions to this specification:
+
+* Becca Hughes
```
Comment (chrisn) [inline index.bs:141]:
```suggestion
* Becca Hughes
* Mounir Lamouri
* Zhiqiang Zhang
```

Comment (chrisn) [reply index.bs:141]: This adds the other people I see in the WICG commit history.

Comment (marcoscaceres) [reply index.bs:141]: Sounds good

**Review by chrisn — APPROVED** [review]
Should we merge? We can always add more in future.

**Pull request merged**